hr_recruitment_security/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,4 @@
+This addon create new group called "Officer (own documents)" in Recruitment.
+
+This new group only show Job positions himself is responsible
+This new group only show Applications himself is responsible
```

Comment:
```suggestion
This new group only shows Applications in which himself is responsible
```

Are you refering to the user? Maybe you could be more clear about who this description is referring to.